Oddities.DIB/Dib.fs

```diff
@@ -1,8 +1,13 @@
-namespace Oddities.Resources
+namespace Oddities
 
 open System
 open System.Buffers.Binary
 
+[<Obsolete("This type has been moved to Oddities.Dib.", false)>]
+type Resources =
+    static member Dib(dib: byte[]) =
+        Dib(dib)
```

Comment:
While this _somewhat_ achieves the goal of making the old type obsolete, it is not the same.

Here, instead of the old type `Dib`, you introduce a static method `Dib`.

Any old code using this type would fall into one of two categories:
- require source changes because it was importing the old type directly or using it by name. For example, consider a variable definition in C#: `Oddities.Resources.Dib _myDib;`
- require recompiling without source changes because it was only using the `Oddities.Resources.Dib` as a constructor. This is only the case for F# code that was written like this: `let dib = Dib somebytes`, and this would _not_ be the case for F# code that was calling the constructor via `let dib = new Dib(somebytes)`. In F#, both syntaxes are valid.

My goal was to not make the users do any of this, so their code would still work without updating the sources or recompilation.